src/components/shared/nextButton.tsx

```diff
@@ -15,7 +20,13 @@ function NextButton(): JSX.Element {
   if (end) {
     return (<div></div>);
   } else {
-    return (<Link to={nextPage}  id='nextButton'>next</Link>);
+    return (
+      <Link to={nextPage}>
+        <button className='nextButton' id={props.isEnabled ?'enabled':''} disabled={!props.isEnabled} onClick={props.onClick}>
```

Comment:
`enabled` shouldn't be an `id`
```suggestion
        <button className={'nextButton' + props.isEnabled ?' enabled':''} disabled={!props.isEnabled} onClick={props.onClick}>
```